src/providers/wallet/wallet.ts

```diff
@@ -1767,7 +1767,8 @@ export class WalletProvider {
       }
 
       const mnemonicHasPassphrase = this.keyProvider.mnemonicHasPassphrase(
-        wallet.credentials.keyId
+        wallet.credentials.keyId,
+        password
```

Comment:
should not be necessary: 
https://github.com/bitpay/bitcore/blob/master/packages/bitcore-wallet-client/src/lib/key.ts#L313